kevlar/call.py

```diff
@@ -304,8 +362,8 @@ def call(targetlist, querylist, match=1, mismatch=2, gapopen=5, gapextend=0,
 def main(args):
     outstream = kevlar.open(args.out, 'w')
     qinstream = kevlar.parse_augmented_fastx(kevlar.open(args.queryseq, 'r'))
-    queryseqs = [record for record in qinstream]
-    targetseqs = [record for record in khmer.ReadParser(args.targetseq)]
+    queryseqs = list(qinstream)
+    targetseqs = list(khmer.ReadParser(args.targetseq))
```

Comment:
No reason to use list comprehension when the more concise `list` casting invokes the generator.